app/controllers/project/settings/donations/payments.js

```diff
@@ -107,24 +113,32 @@ export default Controller.extend({
 
   // assigning a personal id number
 
-  _assignPersonalIdNumber(stripeConnectAccount, personalIdNumber) {
-    set(stripeConnectAccount, 'personalIdNumber', personalIdNumber);
+  _assignLegalEntityPersonalIdNumber(tokenData, stripeConnectAccount) {
+    set(stripeConnectAccount, 'legalEntityPersonalIdNumber', tokenData.id);
 
     return stripeConnectAccount
       .save()
       .then(RSVP.resolve)
       .catch(() => this._wrapError(ACCOUNT_UPDATE_ERROR));
   },
 
+  _creatPersonalIdNumberToken(personalIdNumber) {
```

Comment:
Should we open an issue for an acceptance test that steps through the whole thing?